### PR TITLE
KOGITO-2235: Fixed Scesim artifact generation

### PR DESCRIPTION
**Pull request by yesamer**
@paulovmr A follow up of your previous PR just to fix the Scesim directory as I reported in my mail this morning.

### CODE REVIEW

File: .github/workflows/pull_request_full_downstream_ci.yml [hunk 1]
```diff
@@ -125,7 +125,7 @@ jobs:
         uses: actions/upload-artifact@v1
```
Comment (yesamer) [inline .github/workflows/pull_request_full_downstream_ci.yml:125]: @paulovmr A question regarding line 122: Usually changes on editor are not inside `drools-wb-scenario-simulation-editor-kogito-runtime`, which is only the module which generates the war file. Most of the time changes are inside `drools-wb-scenario-simulation-editor-kogito-client` or `drools-wb-scenario-simulation-editor-client` (but there are other modules to consider). Should we add them as well, in your opinion? Thanks!

Comment (paulovmr) [reply .github/workflows/pull_request_full_downstream_ci.yml:125]: Hi @yesamer , by using the `-am` parameter with this command we include all `drools-wb-scenario-simulation-editor-kogito-runtime` dependencies in the build, so if the runtime depends on those other modules, they will be built as well.

**Review by paulovmr — APPROVED** [review]

File: .github/workflows/pull_request_full_downstream_ci.yml (hunk 1 above)
Comment (tiagobento) [reply .github/workflows/pull_request_full_downstream_ci.yml:125]: Hi @yesamer! The `-am` flag means "also make", which builds every depending module. From `mvn --help`:

```
 -am,--also-make                        If project list is specified, also
                                        build projects required by the
                                        list
```

Comment (yesamer) [reply .github/workflows/pull_request_full_downstream_ci.yml:125]: @paulovmr @tiagobento Thank you both for your synchronized answer! :D

**Pull request merged**